addons/website/static/src/client_actions/website_editor/website_editor.js

```diff
@@ -28,10 +28,10 @@ export class WebsiteEditorClientAction extends Component {
         }, () => [this.props.action.context.params]);
 
         useEffect(() => {
-            this.iframe.el.addEventListener('load', () => {
+            const onPageLoaded = () => {
                 this.currentUrl = this.iframe.el.contentDocument.location.href;
                 this.currentTitle = this.iframe.el.contentDocument.title;
-                history.pushState({}, this.currentTitle, this.currentUrl);
+                history.replaceState({}, this.currentTitle, this.currentUrl);
```

Comment:
Squash with original commit + add comment on why you use replaceState (even if you don't know why :p)